app/models/tutoring_session.rb

```diff
@@ -2,9 +2,11 @@ class TutoringSession < ApplicationRecord
   belongs_to :student
   belongs_to :tutor
 
-  validates :location,     presence: true
-  validates :start_time,   presence: true
-  validates :end_time,     presence: true
+  validates :location, presence: true
```

Comment:
So this model will contain data for a month's worth of sessions but there is only _one_ location? What happens if half of the sessions are in one place and half the sessions are in another?